src/integrationTest/java/uk/gov/hmcts/reform/userprofileapi/integration/RetrieveUserProfileInternalServerErrorIntTest.java

```diff
@@ -81,64 +81,36 @@ public void should_return_500_when_repository_throws_an_unknown_exception() thro
         IdamRegisterUserRequest request = Mockito.mock(IdamRegisterUserRequest.class);
         IdamRegistrationInfo idamRegistrationInfo = new IdamRegistrationInfo(CREATED);
         when(idamService.registerUser(request))
-            .thenReturn(idamRegistrationInfo);
+                .thenReturn(idamRegistrationInfo);
         when(userProfileRepository.findByIdamId(any(String.class)))
-            .thenThrow(new RuntimeException("This is a test exception"));
+                .thenThrow(new RuntimeException("This is a test exception"));
```

Comment:
why the message is "This is a test exception"

Reply:
I have updated the message to reflect the exception being thrown, in this case Runtime Exception, in this commit: https://github.com/hmcts/rd-user-profile-api/pull/249/commits/144afe4b496160730449f10772744c491b34d1cb